[update] fips (#4686)
Kong/docs.konghq.com: src/gateway/kong-enterprise/fips-support.md

content_type: reference
---

> The Federal Information Processing Standard (FIPS) 140-2 standard is a federal standard defined by the National Institute of Standards and Technology that specifies the security requirements that must be satisfied by a cryptographic module. **The core {{site.base_gateway}} product is (FIPS) 140-2 compliant** with the FIPS package that is available for **Ubuntu 20.04**. 

Reviewer: ```suggestion
The Federal Information Processing Standard (FIPS) 140-2 is a federal standard defined by the National Institute of Standards and Technology. It specifies the security requirements that must be satisfied by a cryptographic module. 

**The core {{site.ee_product_name}} product is (FIPS) 140-2 compliant** with the FIPS compliant package that is available for **Ubuntu 20.04**. 
```

> The Federal Information Processing Standard (FIPS) 140-2 standard is a federal standard defined by the National Institute of Standards and Technology that specifies the security requirements that must be satisfied by a cryptographic module. **The core {{site.base_gateway}} product is (FIPS) 140-2 compliant** with the FIPS compliant package that is available for **Ubuntu 20.04**. 

Reviewer 2: ```suggestion
The Federal Information Processing Standard (FIPS) 140-2 standard is a federal standard defined by the National Institute of Standards and Technology that specifies the security requirements that must be satisfied by a cryptographic module. 

**The core {{site.base_gateway}} product is FIPS 140-2 compliant** with the FIPS compliant package that is available for **Ubuntu 20.04**. 
```

> The Federal Information Processing Standard (FIPS) 140-2 standard is a federal standard defined by the National Institute of Standards and Technology that specifies the security requirements that must be satisfied by a cryptographic module. **The core {{site.base_gateway}} product is (FIPS) 140-2 compliant** with the FIPS compliant package that is available for **Ubuntu 20.04**. 

Author: ```suggestion
The Federal Information Processing Standard (FIPS) 140-2 standard is a federal standard defined by the National Institute of Standards and Technology that specifies the security requirements that must be satisfied by a cryptographic module. **The core {{site.base_gateway}} product is (FIPS) 140-2 compliant** with the FIPS package that is available for **Ubuntu 20.04**. 
```

> The {{site.base_gateway}} FIPS 140-2 compliant package is FIPS-compliant **not** "FIPS-validated". Like most enterprise software {{site.base_gateway}} seeks to be FIPS 140-2 compliant. FIPS 140-2 compliance is understood to mean that the software has met all of the rules of FIPS 140-2, but has not been submitted to a NIST testing lab for validation. For FIPS 140-2 compliance, {{site.base_gateway}} replaces the non-FIPS compliant OpenSSL with the FIPS validated library [BoringSSL](https://boringssl.googlesource.com/boringssl/) for encryption. 

Reviewer 3: There's a few sentences here that repeat the word "compliant" in close proximity. Not sure if we want to/can re-write those because I know legal docs don't allow for a lot of wiggle room when it comes to word choice.

Author: done thanks!

> The {{site.base_gateway}} FIPS 140-2 compliant package is FIPS-compliant **not** "FIPS-validated". Like most enterprise software {{site.base_gateway}} seeks to be FIPS 140-2 compliant. FIPS 140-2 compliance is understood to mean that the software has met all of the rules of FIPS 140-2, but has not been submitted to a NIST testing lab for validation. For FIPS 140-2 compliance, {{site.base_gateway}} replaces the non-FIPS compliant OpenSSL with the FIPS validated library [BoringSSL](https://boringssl.googlesource.com/boringssl/) for encryption. 

Reviewer 2: ```suggestion
For FIPS 140-2 compliance, {{site.base_gateway}} replaces the non-FIPS compliant OpenSSL with the FIPS validated library [BoringSSL](https://boringssl.googlesource.com/boringssl/) for encryption. 
```
